c/python/tuple.h

```diff
@@ -71,6 +71,10 @@ class otuple : public oobj {
      */
     void replace(size_t i,  const _obj& value);
     void replace(size_t i,  oobj&& value);
+    virtual ~otuple();
+
+    // Replaces `v` with a copy of `v`, that has `ob_refcnt == 1`.
+    virtual void copy_v();
```

Comment:
This should probably be private? We don't anticipate this function to be used externally

Reply:
Yeah, I've been having some problems making this function virtual and was moving things back and forth, then forgot to make it protected. If we don't want to use virtual functions, we can make it private for sure.